### PR TITLE
Add new variables needed for Neo

### PR DESCRIPTION
**Pull request by lukaspour**
I was trying to use values which were already in `main.tf` as defaults in variables. I had to change `identifier` variable because if there are going to be more RDS instances, the names would be the same.

### CODE REVIEW

**Comment by antonbabenko** [comment]
`local_identifier` was added for the reason like you are describing.
```
# Identifier to append to resources created by modules which do not generate truly unique resources by itself
local_identifier = ""
```

See how it is used in these two cases - [rds-message](https://github.com/TeliaSoneraNorge/devops/tree/master/terraform/telia-no-cpa-stage/eu-west-1/rds-message) and [rds-provision](https://github.com/TeliaSoneraNorge/devops/tree/master/terraform/telia-no-cpa-stage/eu-west-1/rds-provision).

You should use `local_identifier` in modules also.

**Comment by lukaspour** [comment]
I have changed the variable but I am not able the to check the results in `concourse-ci/status`, can I have the access as well?

**Comment by mikael-lindstrom** [comment]
I'll check on how we can add more people but to not leave you waiting the issue this time seems to be formatting. Should be fixed with a simple `terraform fmt`

**Comment by antonbabenko** [comment]
#30 is an automated way of running `terraform fmt`. Run `pre-commit install` once this PR is merged and then commit as usually.

**Comment by mikael-lindstrom** [comment]
https://github.com/TeliaSoneraNorge/telia-terraform-modules/pull/29 has been merged and you should now have access to view the pipeline in concourse

**Comment by lukaspour** [comment]
Thanks, now it seems ok

**Comment by lukaspour** [comment]
@colincoleman is this PR ok? Should I change anything?

File: rds-instance-full/variables.tf [hunk 1]
```diff
@@ -43,6 +43,41 @@ variable "multi_az" {
...
+  description = "DB parameter group"
+  default     = "postgres9.6"
+}
+
+variable "ingress_rule" {
+  description = "Ingress rule to open the ports towards VPC"
+  default     = "postgresql-tcp"
+}
+
 variable "local_environment" {}
```
Comment (antonbabenko) [inline rds-instance-full/variables.tf]: These `local_environment` and below are not needed. They are already inside `main_variables.tf`.

Comment (lukaspour) [reply rds-instance-full/variables.tf]: It got merged from the master, I am going to remove them, thanks

**Comment by lukaspour** [comment]
I guess the problem is with the variables I removed, somehow CI do not use the module as I do in `terraform.tfvars` as `source`.

**Comment by lukaspour** [comment]
@antonbabenko should I un-remove the variables? It worked in CI before.

**Comment by antonbabenko** [comment]
@lukaspour #12 should be reverted (@colincoleman will you do it?). Maybe there were other related/similar PRs also. The idea is to make CI to break for now, but we can use modules as before in our devops repository.

**Comment by lukaspour** [comment]
So, should I change something? I am sorry, but I am stuck on this PR

**Comment by antonbabenko** [comment]
Looks good for now. Merging.

**Pull request merged**